Update index.mdx (#11035)
getsentry/sentry-docs: docs/organization/authentication/sso/okta-sso/index.mdx

1. Click "Save" to create the Sentry app.

> 1. Click "Sign On" to the app settings page to which you are redirected. 

Reviewer: ```suggestion
1. When you're redirected to the app settings page, click "Sign On".
```
